### PR TITLE
Update tutorial-suspicious-activity.md

### PR DESCRIPTION
**Pull request by kremertal**
Note - Starting November 15th, 2022, The Alerts/SMS (text messages) will be deprecated.  Customers who would like to receive text alerts should use Microsoft Power Automate for custom alert automation. For more details ((https://learn.microsoft.com/en-us/defender-cloud-apps/tutorial-suspicious-activity))

### CODE REVIEW

**Comment by dcurwin** [comment]
Docs Build status updates of commit _[c95fc0e](https://github.com/kremertal/CloudAppSecurityDocs/commits/c95fc0e4230fffb8f8a32282a9e7bdb140517f4b)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[CloudAppSecurityDocs/tutorial-suspicious-activity.md](https://github.com/kremertal/CloudAppSecurityDocs/blob/patch-3/CloudAppSecurityDocs/tutorial-suspicious-activity.md) | :warning:Warning | | [Details](#user-content-2a0ffcd893881adf64b618721b59702d-3e337934)

<a id="2a0ffcd893881adf64b618721b59702d-3e337934"></a>
### [CloudAppSecurityDocs/tutorial-suspicious-activity.md](https://github.com/kremertal/CloudAppSecurityDocs/blob/patch-3/CloudAppSecurityDocs/tutorial-suspicious-activity.md)
 - **Line 138, Column 133**: **[Warning: hard-coded-locale - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://learn.microsoft.com/en-us/defender-cloud-apps/tutorial-suspicious-activity' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/daf9759b-aa27-219c-4b5e-53f1b72a2d34/202209211608193731-592/BuildReport?accessString=5f4fe59f8ac78c341ab89aae46303e1dff0a74518e8de65ccd5bced6c99770e5).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/daf9759b-aa27-219c-4b5e-53f1b72a2d34?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings or suggestions unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by jborsecnik** [comment]
@dcurwin, 

Can you review the proposed changes? Note that the the "en-us" would have to be removed from the new URL.

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: CloudAppSecurityDocs/tutorial-suspicious-activity.md [hunk 1]
```diff
@@ -134,6 +134,9 @@ To prevent alert fatigue, set the **daily alert limit**.
 
 ## Phase 5: Configure alerts
 
+Note - Starting November 15th, 2022, The Alerts/SMS (text messages) will be deprecated. 
+Customers who would like to receive text alerts should use Microsoft Power Automate for custom alert automation. For more details ((https://learn.microsoft.com/en-us/defender-cloud-apps/tutorial-suspicious-activity))
```
Comment (dcurwin) [inline CloudAppSecurityDocs/tutorial-suspicious-activity.md:138]:
```suggestion
> If you would like to receive text alerts, you should use Microsoft Power Automate for custom alert automation. For more information, see [Integrate with Microsoft Power Automate for custom alert automation](flow-integration.md).
```

**Comment by dcurwin** [comment]
Docs Build status updates of commit _[29456f1](https://github.com/kremertal/CloudAppSecurityDocs/commits/29456f1f0a1944a9b820490778c6ecca66f27aa7)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[CloudAppSecurityDocs/tutorial-suspicious-activity.md](https://github.com/kremertal/CloudAppSecurityDocs/blob/patch-3/CloudAppSecurityDocs/tutorial-suspicious-activity.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/daf9759b-aa27-219c-4b5e-53f1b72a2d34/202209220948340295-592/BuildReport?accessString=f918a758535f33aac66b928eae1abfcd53e7b3e3e343d9490f436e1fd7e645c7).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/daf9759b-aa27-219c-4b5e-53f1b72a2d34?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**